app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def show; end
+
+  def posts; end
+
+  def post; end
```

Comment:
- [ ] We do not have to add these two actions here because what we need to be in this UsersController for now is just the `index` and `show` actions. Kindly remove `posts` and `post` actions 😃

- [ ] Kindly generate PostsController and create the `index` and `show` actions. Maybe the `posts` and `post` actions here are what you wanted to use, but that's not how it works okay, let's create a new controller for Posts and add the `index` and `show` actions.